client/src/shapes/helpers.ts

```diff
@@ -244,3 +245,14 @@ export const normalizeBoundingBox = (bb: BoundingBox): BoundingBox => ({
   width: Math.abs(bb.width),
   height: Math.abs(bb.height),
 });
+
+export const toBorderRadiusArray = (
+  borderRadius: BorderRadius['borderRadius'],
+): BorderRadiusArrayValue => {
+  if (!borderRadius) {
```

Comment:
nit: `borderRadius === 0` is clearer imo but achieves the same thing so no biggie